lab2.js

```diff
@@ -100,16 +98,40 @@ function sayHello (sb) {
     // to do the translating
 
     //TODO: put this on the SentientBeing prototype
-  }
 
 // TODO: create three SentientBeings, one for each language in the
 // 'hello' object above.
-var klingon = new SentientBeing(); // TODO: make a klingon
-var romulan = new SentientBeing(); // TODO: make a romulan
-var human = new SentientBeing(); // TODO: make a human
+
+var hello = {
+  klingon: 'nuqneH',
+  romulan: 'Jolan\'tru',
+  'federation standard': 'hello'
+};
+
+function SentientBeing(homePlanet, language) {
+  this.homePlanet = homePlanet;
+  this.language = language;
+}
+
+// sb is a SentientBeing object
+SentientBeing.prototype.sayHello = function(sb) {
+  console.log(hello[this.language]);
+  return (hello[sb.language]);// this is the only place its breaking,
+};
+
+SentientBeing.prototype.sayHello();
+
+var klingon = new SentientBeing('Qo\'noS', 'klingon', 'nuqneH'); // TODO: make a klingon
```

Comment:
the last argument will just be ignored here, since SentientBeing only takes two parameters